src/store/user/actions.ts

```diff
@@ -0,0 +1,10 @@
+import { UsersActionTypes, UserType, AddUserAction } from './types';
+
+export const addUserAction = (userData: UserType): AddUserAction => ({
```

Comment:
nit: I find the naming to misleading. This is used for login, not registration, so you're not "adding" the user to any database (aside from the store). Same goes for courses and authors, to a lesser degree - you're calling "add courses" during the initial App.tsx load. I'd go with "setUser" or "storeUser," etc.

Reply:
I changed the naming for courses, authors and user : from add... to store...